sklearn/preprocessing/tests/test_target.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+
+from sklearn.utils.testing import assert_array_almost_equal
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_raises_regex
+from sklearn.utils.testing import assert_allclose
+
+from sklearn.preprocessing import TransformTargetRegressor
+
+from sklearn.linear_model import LinearRegression
+from sklearn.preprocessing import StandardScaler
+
+from sklearn import datasets
+
+friedman = datasets.make_friedman1(random_state=0)
```

Comment:
Can you add a test that it really is the identity transform with no params set,  because the behavior is not explicit in the code, but delegated to FunctionTransformer.